### PR TITLE
Listrak/v1

### PR DESCRIPTION
**Pull request by ajmers**
This PR adds documentation for the Listrak integration.

### CODE REVIEW

**Comment by erinkcochran87** [comment]
@KAllan357 The rendered preview [is available here](https://deploy-preview-84--stitchdocs.netlify.com/docs/integrations/saas/listrak).

File: _saas-integrations/listrak/listrak-latest.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+certified: false
+
+historical: "1 year"
+frequency: "30 minutes"
+tier: "Free"
+status-url: 
+icon: /images/integrations/icons/listrak.svg
+whitelist:
+  tables: true
+  columns: false
```
Comment (KAllan357) [inline _saas-integrations/listrak/listrak-latest.md]: I'm not aware of how we can confirm (as we don't have a test account), but I think we do support propert-selection here so this should be true I think.

**Comment by KAllan357** [comment]
👍

File: _saas-integrations/listrak/listrak-latest.md (hunk 1 above)
Comment (erinkcochran87) [reply _saas-integrations/listrak/listrak-latest.md]: Isn't there something that's added to the tap itself to enable field selection?

Comment (KAllan357) [reply _saas-integrations/listrak/listrak-latest.md]: Yes. Upon closer inspection, it looks like everything is just set to "Automatic" so I think this is okay.

**Pull request merged**